### PR TITLE
Modified versions and refactored code

### PR DESCRIPTION
**Pull request by MalshaL**
## Purpose
Modified versions and refactored code.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,4 +1,21 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (chanikag) [inline pom.xml]: Wrong license header?

File: samba/pom.xml [hunk 2]
```diff
@@ -1,17 +1,34 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (chanikag) [inline samba/pom.xml]: Wrong license header?

File: pom.xml (hunk 1 above)
Comment (MalshaL) [reply pom.xml]: Fixed in #14.

File: samba/pom.xml (hunk 2 above)
Comment (MalshaL) [reply samba/pom.xml]: Fixed in #14.

**Pull request merged**